docs/language/lateral-subqueries.md

```diff
@@ -116,12 +116,12 @@ parenthesized form:
 > dataflow operator.
```

Comment:
```suggestion
> pipeline operator.
```

Reply:
@nwt: It was actually intentional that I didn't touch this one because I'd already posed a question about it at https://github.com/brimdata/zed/pull/5264/files#r1747648931 and so expected to change it there based on what reviewers said. In essence, I feel like this text needs more help than just the dataflow->pipeline change, and perhaps the text should be removed entirely. If it stays, I'd certainly make sure it says "pipeline" regardless. Could you offer your take on if the overall statement is fine as is, and if not, how it could be clarified or if it should be removed?

Reply:
This comment has my take: https://github.com/brimdata/zed/pull/5264#discussion_r1761003157.